app/enterprise/0.32-x/upgrades-migrations.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Upgrades and Migrations
+---
+
+# Upgrades and Migrations
+## Upgrades of Kong EE
+
+We define an upgrade, the process of changing the version of the software currently deployed to a more recent one, there is no difference with concept of update; normally associated with an upgrade between minor versions.
+
+We suggest to follow the below steps to perform an effective upgrade, this approach will require downtime, in case you need high availability review the Blue/Green Deployment section.
+
+1. Download the destination version of the product you are interest to upgrade to in Bintray
+2. Take a backup copy of the production database
+3. Review the [changelog](https://docs.konghq.com/enterprise/changelog/) for the destination version of the upgrade
```

Comment:
`3. Review the [changelog](https://docs.konghq.com/enterprise/changelog/) for the version you are upgrading to`

Reply:
Should we place the changelog review in the 1st step or 2nd - IMHO people should review the changelog before any further steps.

Reply:
Due logical order if a customer doesn't have access to Bintray is useless to review change log, therefore I will keep it as step 3.
Step 2 should remain "copy the DB" in my opinion as order of importance.